src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+const { getOne: getOneUser } = require('../services/user.service');
+
+const get = async(req, res) => {
+  const userId = parseInt(req.query.userId);
+  const { categories, from, to } = req.query;
+
+  const filteredExpenses = await expensesService.filter(
+    userId,
+    categories,
+    from,
+    to,
+  );
+
+  res.json(filteredExpenses);
+};
+
+const create = async(req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+  const user = await getOneUser(userId);
+
+  if (
+    !userId
+    || typeof spentAt !== 'string'
+    || !title
+    || !amount
+    || !user
```

Comment:
`category` is not required?

Reply:
@Andriy-Shymkiv one test has an product without category and a requirement to create an expense without one field, idk how to solve this in another way so I'd break the rule of the API documentation about category :) how can I solve it properly?
![image](https://github.com/mate-academy/node_accounting-app-with-db/assets/107129593/b0229ed5-3dac-4e59-9bc2-e40fb7d77f28)

Reply:
![image](https://github.com/mate-academy/node_accounting-app-with-db/assets/107129593/4d6b0201-95df-4b18-b2a3-eccaa2a4fd03)